src-electron/db/query-attribute.js

```diff
@@ -297,24 +265,13 @@ async function selectAttributeDetailsFromEnabledClusters(
       define: x.DEFINE,
       mfgCode: x.MANUFACTURER_CODE,
       isWritable: x.IS_WRITABLE,
-      isNullable: x.IS_NULLABLE,
```

Comment:
It'll also be useful to understand the speed up figures as well.

Reply:
Initially the idea was to give more information for the template creators. However, this has a cost on performance because nested helpers call the queries multiple times decreasing the performance. So adding what is required on a use by basis is better in my regard for performance. I added cluster Id back.

Since the generation time has been different on each run by a few seconds locally it has been difficult to get a stats consistently. However I can try to come up with a few stats on multiple runs with/without my changes.